src/components/menuBar/menuBar.tsx

```diff
@@ -11,20 +13,15 @@ const MenuBarContent = styled.div`
   box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.04), 0px 2px 6px rgba(0, 0, 0, 0.04), 0px 8px 24px rgba(0, 0, 0, 0.06);
 `;
 
-const NormalButtonContainer = styled.button`
-  line-height: 17px;
-  letter-spacing: -0.02em;
-  padding: 6px 12px;
-  border-radius: 0.5rem;
-`;
-
 const SearchBar = styled.div`
   border: 1px solid #e2e7eb;
   box-sizing: border-box;
 `;
 
 export const MenuBar: React.FunctionComponent = () => {
   const [search, setSearch] = useState<string>("");
+  const [selectedButton, setSelectedButton] = useState<Tag | null>(null);
```

Comment:
You may use undefined in this case. When a variable can be undefined, just add a `?` behind.

```suggestion
  const [selectedButton, setSelectedButton] = useState<Tag?>();
```